lib/reservation_manager.rb

```diff
@@ -0,0 +1,143 @@
+require "pry"
+require "date"
+
+class Reservation_Manager
```

Comment:
Minor style thing I'm not sure we mentioned in class.  When a class is multiple words you just use capital letters and not underscores.

```suggestion
class ReservationManager
```